ipykernel/_version.py

```diff
@@ -1,5 +1,16 @@
-version_info = (5, 0, 0, 'dev')
-__version__ = '.'.join(map(str, version_info))
+version_info = (5, 0, 0, 'b1')
+__version__ = '.'.join(map(str, version_info[:3]))
+
+# pep440 is annoying, beta/alpha/rc should _not_ have dots or pip/setuptools
```

Comment:
This isn't an issue because we are using setuptools in setup.py. It only comes up where we skip the setuptools import in setup.py when making an sdist, which we don't do here.